database/secret_test.go

```diff
@@ -8,10 +8,15 @@ import (
 	"database/sql"
 	"reflect"
 	"testing"
+	"time"
 
 	"github.com/go-vela/types/library"
 )
 
+var currentTime time.Time = time.Now()
+var tsCreate string = currentTime.Format(time.UnixDate)
+var tsUpdate string = currentTime.Add(time.Hour * 1).Format(time.UnixDate)
```

Comment:
I think you can skip providing the underlying type of the variable if you're assigning it a value.

Also, this one is a bit nit-picky but could we update this to be a single `var` block?

```suggestion
var (
    currentTime = time.Now()
    tsCreate = currentTime.Format(time.UnixDate)
    tsUpdate = currentTime.Add(time.Hour * 1).Format(time.UnixDate)
)
```

Reply:
Yeah for sure! On this, I was informed that we are recording timestamps in our build object as ints. [https://github.com/go-vela/types/blob/911ffbea2e427883ccca2cc3f3769fdae75e0946/database/build.go#L42-L43](url). Example of utilization: [https://github.com/go-vela/server/blob/88520ca76a0856316b6c17d15d8305198d603735/api/build.go#L141](url). I will update that to be consistent.